app/javascript/packs/webauthn-setup.js

```diff
@@ -22,7 +22,7 @@ function webauthn() {
   }
   const continueButton = document.getElementById('continue-button');
   continueButton.addEventListener('click', () => {
-    document.getElementById('spinner').className = '';
+    document.getElementById('spinner').classList.remove('hidden');
```

Comment:
do we need to add the `classList` polyfill to `loadPolyfills` at the bottom?

Reply:
I discovered recently that most simple use of `classList` is actually implemented in IE11. The main one that isn't supported is calling `classList#toggle` with a second argument. This one's fine though.

https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/remove#browser_compatibility

Reply:
Oh right right, `.add` and `.remove` are fine, this sounds familiar